.vscode/settings.json

```diff
@@ -1,4 +1,8 @@
 {
   "typescript.preferences.quoteStyle": "single",
-  "css.lint.unknownAtRules": "ignore"
+  "css.lint.unknownAtRules": "ignore",
+  "editor.formatOnSave": false,
```

Comment:
https://eslint.style/guide/faq#how-to-auto-format-on-save

Reply:
can we get rid of the on save one? I prefer my editor not do that personally.

Reply:
Oh yeah, totally.

Reply:
Happy to not do this at all.

Reply:
reverted this